integration/delimited-methods/delimited-methods-test.ts

```diff
@@ -0,0 +1,26 @@
+import { Simple } from './delimited-methods';
+
+describe('delimited-methods', () => {
+  const message: Simple = {
+    age: 42,
+    name: 'John Doe',
+    children: ['Jane', 'Jack', 'Joe'],
+  };
+
+  it('encodes with a length delimiter', () => {
+    const encoded = Simple.encodeDelimited(message).finish();
+
+    // -1 for the length delimiter
+    const length = encoded.length - 1;
+    const delimiter = encoded[0];
+
+    expect(length).toEqual(delimiter);
+  });
+
+  it('decodes length-delimited messages', () => {
+    const encoded = Simple.encodeDelimited(message).finish();
+    const decoded = Simple.decodeDelimited(encoded);
+
+    expect(decoded).toEqual(message);
+  });
```

Comment:
How about changing this test (or adding one) to encode and decode 2 messages?

It will give us more confidence that the main use case is working, and also demonstrate how to use the feature in practice.

Reply:
Definitely! Just added streaming tests via 0ec5e53, please let me know what do you think. Thanks!